### DISCUSSION TITLE
channel is already in the process of being force closed[bug]:

### DISCUSSION THREAD

**Discussion by byjlw** [lightningnetwork/lnd#8410] [Troubleshooting] [answered]
### Background
New nodes. 
Set up two nodes and funded them
Tried to establish a channel but set the sat_per_vbyte value to 5 (way too low) accidentally. 
Because it was low it was sitting around there with no confirmations for a long time
I ran a closechannel --force close on it
Now it's been just sitting in limbo and anytime I try to force close i get "channel is already in the process of being force closed"

### Your environment

* 0.17.3-beta 
* Windows 11 pro
* qt 5.15.5
* any other relevant environment details

### Steps to reproduce

Set up two nodes and fund them
Set up a channel from 5 to b with sat_per_vbyte=5
closechannel <id> --force 

### Expected behaviour

channel would close

### Actual behaviour
"channel is already in the process of being force closed"
This still happens after 24 hours
Now it's been just sitting in limbo and anytime I try to force close i get "channel is already in the process of being force closed"

TransactionID: 4249b7bb841c9128298f0e34a550ce3407be45317b7b5a8230f8dffb05837d90

**Comment by alexbosworth** [comment, marked as answer]
You could try the reset-wallet-transactions flag and double spend the original utxo?

**Comment by byjlw** [comment]
Yeah double spending utxo "should" fix the problem by invalidating the unconfirmed transaction. I'll try it. That said, getting into this state is easy and the messages aren't clear. While in this state no one can open a channel to this node.
